test/server.js

```diff
@@ -16,7 +16,7 @@ var coap      = require('../')
   , sinon     = require('sinon')
   , params    = require('../lib/parameters')
 
-describe('server', function() {
+describe.only('server', function() {
```

Comment:
Because I forgot to remove it :D Same goes for the "TODO". God bless Code Reviews.